src/datatypes/colorizer.md

```diff
@@ -59,7 +62,8 @@ Services report errors that try to use a logarithmic gradient specification with
     }
   ],
   "noDataColor": [0, 0, 0, 0],
-  "defaultColor": [0, 0, 0, 0]
+  "overColor": [255, 0, 0, 255],
+  "underColor": [0, 0, 255, 255]
```

Comment:
Same here.

Maybe you should update the description above. You could have a focus on a certain area, e.g., to highlight differences in temperature, but you declear everything above 20° just as hot and everything below 0 just as cold. But you can be very distinctive between 0 and 20°.

Reply:
I tried to come up with a reasonable setting now and hopefully more representative colors.